client/components/auth/SigninForm.jsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import { PropTypes } from 'prop-types';
+import requireauth from '../../utils/requireAuth';
+import { login } from '../../actions/authActions';
+import validateInput from '../../../server/shared/validations/signin';
```

Comment:
It is a very good practice to import file the way it is exported which often demonstrate how a file is saved. This file is saved under validations folder as signin and you have imported as validateInput, I think you should really consider this.